programmers/로또의-최고-순위와-최저-순위.js

```diff
@@ -0,0 +1,38 @@
+function rankswitch(n) {
+    switch(n) {
+            case 0 :
+                return 6
+                break
+            case 1 : 
+                return 6
+                break
+            case 2 : 
+                return 5
+                break
+            case 3 : 
+                return 4
+                break
+            case 4 : 
+                return 3
+                break
+            case 5 : 
+                return 2
+                break
+            default :
+                return 1
+            }
+}
+
+function solution(lottos, win_nums) {
+    let matchednum = 0 //확인된 번호중 맞는 번호 갯수
+    let unknown = 0 //미확인번호 갯수
+    
+    for(let lottonum of lottos) {
+        if(win_nums.indexOf(lottonum) !== -1) matchednum += 1 //맞는번호가있을때 
+        else if(lottonum === 0) unknown += 1 //모르는번호가 있을때(0이 있을때)
+    }  
+    let best = matchednum + unknown
+    let worst = matchednum 
+    const answer = [rankswitch(best), rankswitch(worst)]
+    return answer 
```

Comment:
1. 변수는 상단에 선언 하는것이 좋습니다.
2. 변수명, 함수명 은 케멀케이스를 따라주세요
ex ) matchnum -> matchNum
3. 세미콜론 붙여주세요~
4. for , if , else if 등 뒤에 공백을 붙이셔야 합니다. 
5.  굳이 answer 담아서 리턴하지 말고 리스트를 그대로 return하세요
6. indexOf는 없으면 -1을 반환해서 존재하지 않는지 알 수 있는건데 true false를 반환하는 includes 쓰는게 좋습니다.
```suggestion
    let matchedNum = 0; //확인된 번호중 맞는 번호 갯수
    let unknown = 0;//미확인번호 갯수
    let best = matchedNum + unknown;
    let worst = matchedNum;
 
    
    for (let lottoNum of lottos) {
        if (win_nums.includes(lottoNum) !== -1) matchedNum += 1; //맞는번호가있을때 
        else if (lottoNum === 0) unknown += 1; //모르는번호가 있을때(0이 있을때)
    }  
   
    return [rankswitch(best), rankswitch(worst)] ;
```